lms/djangoapps/certificates/tests/test_cert_management.py

```diff
@@ -252,3 +253,61 @@ def _mock_passing_grade(self):
         with patch(symbol) as mock_grade:
             mock_grade.return_value = {'grade': 'Pass', 'percent': 0.75}
             yield
+
+
+@attr('shard_1')
+class RescindCertificatesTest(CertificateManagementTest):
+    """
+    Tests for rescinding certificates.
+    """
+    command = rescind_cert
+
+    def setUp(self):
+        """
+        We just need one course here.
+        """
+        super(RescindCertificatesTest, self).setUp()
+        self.course = self.courses[0]
+        self.key = self.course.location.course_key
+        self._create_cert(self.key, self.user, CertificateStatuses.downloadable)
+
+    def test_rescind_cert_success(self):
+        call_command(
+            'rescind_cert',
+            None,
+            course=unicode(self.key),
+            username=self.user.username,
+        )
+        certificate = GeneratedCertificate.objects.get(
+            user=self.user,
+            course_id=self.key
+        )
+        self.assertEqual(certificate.status, CertificateStatuses.unavailable)
+
+    def test_rescind_cert_status_change(self):
+        self._run_command(course=unicode(self.key), username=self.user.username, status=CertificateStatuses.notpassing)
```

Comment:
Given the length, this should probably be split into multiple lines, mirroring what you did above in `test_rescind_cert_success`'s call to `call_command`.